tools/cli/help.txt

```diff
@@ -1,19 +1,25 @@
 >>>
 (the first line of each entry is the summary for autogenerated command lists)
-Usage: meteor [--release <release>] [--help] <command> [args]
-       meteor help <command>
+Usage: meteor [--release <release>] [--help] <built-in command> [args]
+       meteor help <built-in command>
+       meteor <external command> [args]       
```

Comment:
Right I'm just saying that putting "npm" in there makes sense since that's where these commands come from. Or maybe just saying more in the docs is enough like "get more commands by npm installing them"